new-log-viewer/src/services/decoders/JsonlDecoder/utils.ts

```diff
@@ -0,0 +1,100 @@
+import dayjs, {Dayjs} from "dayjs";
+import utc from "dayjs/plugin/utc";
+
+import {
+    JsonObject,
+    JsonValue,
+} from "../../../typings/js";
+import {
+    INVALID_TIMESTAMP_VALUE,
+    LOG_LEVEL,
+} from "../../../typings/logs";
+
+
+// eslint-disable-next-line import/no-named-as-default-member
+dayjs.extend(utc);
+
+/**
+ * A log event parsed from a JSON log.
+ */
+interface JsonLogEvent {
+    timestamp: Dayjs,
+    level: LOG_LEVEL,
+    fields: JsonObject
+}
+
+/**
+ * Narrow JSON value to JSON object if compatible.
+ * Reference: https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates
+ *
+ * @param fields
+ * @return Whether type is JsonObject.
+ */
+const isJsonObject = (fields: JsonValue): fields is JsonObject => {
+    return "object" === typeof fields;
+};
+
+/**
+ * Converts JSON log level field into a log level value.
+ *
+ * @param logLevelField
+ * @return Integer for log level.
+ */
+const convertToLogLevelValue = (logLevelField: JsonValue | undefined): number => {
```

Comment:
How about:

* `convertToLogLevel`
* `logLevelField` -> `field` since it may be any field, we're just trying to convert it into a log level.

Reply:
Did you miss the function rename suggestion?